app/src/userpages/components/StreamPage/Show/HistoryView/index.jsx

```diff
@@ -78,7 +80,7 @@ class HistoryView extends Component<Props, State> {
     }
 
     mounted = false
-    fileUploadRef = React.createRef()
+    fileUploadRef: Ref<Dropzone> = React.createRef()
```

Comment:
This one is interesting. We have to declare what does `FileUpload` assign to this ref. In this case it's a `Dropzone` instance.